community/2023-11-29-community-meeting.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 2023-11-29 Community Meeting
+authors: [brooks]
+tags: [community, meeting]
+description: Agenda, notes, and recording for the 2023-11-29 community meeting
+---
+
+import ReactPlayer from 'react-player/youtube';
+
+### Agenda
+
+- 
+- 
+- 
+
+<!--truncate-->
```

Comment:
```suggestion
{/* truncate */}
```
File needs to be mdx, also.

Reply:
Hey I made a PR to fix both of those things (for good, hopefully) here: https://github.com/wasmCloud/wasmcloud.com-dev/pull/219